docs/ff-concepts/localization-accessibility/localization.md

```diff
@@ -0,0 +1,221 @@
+
+To add multi-language support in FlutterFlow, navigate to **Settings and Integrations** > **Languages**, add the languages to support, set a primary language as a fallback, and optionally choose a display language. Then, use **Translate All** for automatic translations and adjust them if needed. Finally, verify translations on different pages by changing the language dropdown in the canvas.
+
+:::warning
+
+Changing the primary language after translating all of your text will clear the existing translations for other languages.
+
+:::
+
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/po8PphcS06ljCvKsTG76?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+## LanguageSelector Widget
+
+The **LanguageSelector** widget in FlutterFlow allows users to switch to their preferred language in real-time without needing to restart the app. It displays the currently selected language and, when interacted with, presents a list of all available languages for easy selection.
+
+It's particularly useful on onboarding screens or within settings menus to allow users to customize their language preferences.
+
+### LanguageSelector Properties
+
+You can customize the appearance using the various properties available under the Properties Panel.
+
+![language-selector-properties.avif](imgs/language-selector-properties.avif)
+
+:::tip
+
+By default, the LanguageSelector widget does not persist the user's language choice across app sessions. To retain the selected language, enable the **Persist Selection** option under language settings.
+
+:::
+
+## Set App Language Manually [Action]
+
+Sometimes, you might prefer not to use the default [LanguageSelector widget](#languageselector-widget) and instead implement a custom widget for language switching. For example, you could create a custom language selection screen that appears when the app first launches.
+
+You can use the **Set App Language** action to let users choose their preferred language from the available options.
+
+:::info
+
+Note that this action affects only the app's language and does not modify the device's system language.
+
+:::
+
+![set-app-lang-action.png](imgs/set-app-lang-action.png)
+
+## Managing Translation
+
+There are two ways you can manage the app text translation:
+
+### Inside Language Settings
```

Comment:
This can be bullet points itself? Or just bold headings will work. No need for headers.